File: aoc2020/src/bin/all.rs
fn main() {
    println!("🎄 Advent of Code 2020 🎄\n");
    println!("Compiling 🛠️\n");
    (1..=25).for_each(|day| {
        let cmd = std::process::Command::new("cargo").args(&["run", "--release", "--bin", &format!("day{:02}", day)]).output().unwrap();
        let out = String::from_utf8(cmd.stdout).unwrap();
        println!("> Day {}\n{}", day, out);
    })
}

File: aoc2017/src/all.py
import os

print("🎄 Advent of Code 2017 🐍\n")
for day in range(1, 26):
    print(f"> Day {day}")
    day = day if day >= 10 else f"0{day}"
    os.system(f"python src/day{day}.py")
    print()
